include/aspect/postprocess/visualization/strain_rate_tensor.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (C) 2015 - 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_postprocess_visualization_strain_rate_tensor_h
+#define _aspect_postprocess_visualization_strain_rate_tensor_h
+
+#if DEAL_II_VERSION_GTE(9,0,0)
```

Comment:
This doesn't compile (and same for the .cc file):
```
/home/dealii/source/include/aspect/postprocess/visualization/strain_rate_tensor.h:25:24: error: missing binary operator before token "("
 #if DEAL_II_VERSION_GTE(9,0,0)
                        ^
```
The reason is that it doesn't know about that macro. You'll have to move this `#if` statement *after* the `#include` files for this to work.